src/proxy/go/go.c

```diff
@@ -41,7 +42,7 @@
  *
  *      - name: shortname of the plugin.
  *      - description: plugin description.
- *      - type: input, output, filter, whatever.
+ *      - type: input, output, filter, processor, custom.
```

Comment:
I am not sure if processors can be written in Go today cc: @cosmo0920

Reply:
Currently, we can't do that. If processors can be written in Go, there is huge opportunity to handle various type of payloads including otel payloads which is belongs to traces type of events.